docs/server/push-duty.rst

```diff
@@ -40,6 +40,40 @@ This way we can ensure that:
 Making multiple releases of a package during a weekly milestone is totally
 fine since this helps with testing smaller sets of changes.
 
+Security Fixes
+++++++++++++++
+
+Security fixes for addons-server live on a separate private repository:
+
+    * `addons-server-security <https://github.com/mozilla/addons-server-security>`_
+
+When making a pull request against this repository, the remote branch should
+not be published to one's fork but to the repository itself, keeping the
+changes entirely private. Once the PR has been reviewed, it should *not* be
```

Comment:
Ah, I wasn't aware you could do that with a normal account. In any case it's best to keep the remote branches on mozilla/addons-server-security to make merging them easier (no need for extra commands to fetch the author's remote or extra config)